.github/styles/Vocab/Base/accept.txt

```diff
@@ -0,0 +1,109 @@
+github
```

Comment:
This is the same as https://github.com/cloudquery/cloudquery/tree/bac87adb620104eb4eb7fcc715cfe6c44c96f8f5/.github/styles.

As a follow up we could create a dedicated repository just for those files and publish the styles as a `zip` artifact, then use a URL instead of a directory